services/gateway/util/src/main/java/org/eclipse/ditto/services/gateway/util/config/security/DefaultSubjectIssuerConfig.java

```diff
@@ -0,0 +1,123 @@
+
+import org.eclipse.ditto.model.policies.SubjectIssuerConfig;
+import org.eclipse.ditto.services.utils.config.ConfigWithFallback;
+import org.eclipse.ditto.services.utils.config.KnownConfigValue;
+
+public class DefaultSubjectIssuerConfig implements SubjectIssuerConfig {
+
+    private final String issuer;
+
+    private final List<String> authSubjectTemplates;
+
+    private DefaultSubjectIssuerConfig(final ConfigWithFallback configWithFallback) {
+        issuer = configWithFallback.getString(SubjectIssuerConfigValue.ISSUER.getConfigPath());
+        authSubjectTemplates = configWithFallback.getStringList(SubjectIssuerConfigValue.AUTH_SUBJECTS.getConfigPath());
+    }
+
+    private DefaultSubjectIssuerConfig(final String issuer, final List<String> authSubjectTemplates) {
+        this.issuer = issuer;
+        this.authSubjectTemplates = authSubjectTemplates;
+    }
+
+    public static DefaultSubjectIssuerConfig of(final Config config, final String issuerName) {
+        // TODO: issuerName should not be needed here.
+        return new DefaultSubjectIssuerConfig(
+                ConfigWithFallback.newInstance(config, issuerName, new KnownConfigValue[0]));
+    }
+
+    /**
+     * Returns a new SubjectIssuerConfig based on the provided strings.
+     *
+     * @param issuer       the issuer's endpoint {@code issuer}.
+     * @param authSubjectTemplates list of authorizationsubject placeholder strings
+     *                     {@code authSubjectTemplates}.
+     * @return a new SubjectIssuerConfig.
+     * @throws NullPointerException     if {@code issuer} or {@code authSubjectTemplates} is
+     *                                  {@code null}.
+     * @throws IllegalArgumentException if {@code issuer} or {@code authSubjectTemplates} is
+     *                                  empty.
+     */
+    public static DefaultSubjectIssuerConfig of(final String issuer, final List<String> authSubjectTemplates) {
+        checkNotNull(issuer, "issuer");
+        argumentNotEmpty(authSubjectTemplates, "authSubjectTemplates");
+
+        return new DefaultSubjectIssuerConfig(issuer, authSubjectTemplates);
+    }
+
+    public final String getIssuer() {
+        return issuer;
+    }
+
+    public final List<String> getAuthorizationSubjectTemplates() {
+        return authSubjectTemplates;
+    }
+
+    @Override
+    public boolean equals(final Object o) {
+        if (this == o) {
+            return true;
+        }
+        if (o == null || getClass() != o.getClass()) {
+            return false;
+        }
+        final DefaultSubjectIssuerConfig that = (DefaultSubjectIssuerConfig) o;
+        return Objects.equals(issuer, that.issuer) && authSubjectTemplates.equals(that.authSubjectTemplates);
+    }
+
+    @Override
+    public int hashCode() {
+        return Objects.hash(issuer, authSubjectTemplates);
+    }
+
+    @Override
+    @Nonnull
+    public String toString() {
+        return issuer + authSubjectTemplates;
+    }
```

Comment:
```suggestion
    @Override
    public String toString() {
        return getClass().getSimpleName() + " [" +
                "issuer=" + issuer +
                ", authSubjectTemplates=" + authSubjectTemplates +
                "]";
    }
```
That toString() does not make too much sense - concatenating a string with a List.
Suggesting our "default" toString() here..